src/main/java/com/fakng/fakngagrgtr/configuration/ParserConfig.java

```diff
@@ -26,4 +29,13 @@ public WebClient webClient() {
                 .exchangeStrategies(strategies)
                 .build();
     }
+
+    @Bean
+    public ObjectMapper objectMapper() {
+        return Jackson2ObjectMapperBuilder
+                .json()
+                .build()
+                .configure(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES, false);
```

Comment:
Nice one. It'll allow us not to write annotations all the time

Reply:
Yep:)